src/ploomber/cli/report.py

```diff
@@ -14,8 +16,16 @@ def main():
             '-o',
             help='Where to save the report, defaults to pipeline.html',
             default='pipeline.html')
+        parser.add_argument(
+            '--backend',
+            '-b',
+            help=('How to generate the plot: d3 or '
+                  'pygraphviz. Using whatever is '
+                  'available by default'),
```

Comment:
> How to generate the plot: d3 or pygraphviz. Using whatever is available by default

Maybe Decides on the backend to generate the plot: d3 or pygraphviz, the one available by default.

Reply:
Updated